micromanager_gui/_core_widgets/_presets_widget.py

```diff
@@ -125,6 +127,8 @@ def refresh(self) -> None:
                 presets = self._mmc.getAvailableConfigs(self._group)
                 self._combo.addItems(presets)
                 self._combo.setEnabled(True)
+                self._combo.setCurrentText(self._mmc.getCurrentConfig(self._group))
+                self._set_if_props_match_preset()
```

Comment:
I think you can just do this?

```suggestion
                self._set_if_props_match_preset()
```

Reply:
@fdrgsp bump here. I was looking at the channel widget PR and arrived back here.